lib/std/os/bits/linux.zig

```diff
@@ -858,6 +858,14 @@ pub const sockaddr_in = extern struct {
     port: in_port_t,
     addr: u32,
     zero: [8]u8,
+    pub fn init(family: sa_family_t, port: in_port_t, addr: u32) @This() {
```

Comment:
This would be the first "method" added to an "OS bits" struct, which in C is plain old data. I think it will be better to leave all of these as data types only. The higher level abstraction goes elsewhere; not in std.os.

Reply:
ok

Reply:
moved everything into `net.zig`, didn't cover all the OS cases but I figure those will get filled in as we need them.